### PR TITLE
compile under LTS-7.13

### PR DESCRIPTION
**Pull request by AlexeyRaga**
## Changes

- Make it compile under LTS 7.13
- LTS 8.3 will even have buffers

### CODE REVIEW

File: stack.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+# This file was automatically generated by 'stack init'
```
Comment (joshgodsiff) [inline stack.yaml:1]: Do we actually want to add the stack.yaml?

Comment (AlexeyRaga) [reply stack.yaml:1]: Yes, I think so. It doesn't hurt if you still want to use cabal, and there is an open issue against the parent repo to support stack...

Comment (joshgodsiff) [reply stack.yaml:1]: Fair enough

**Comment by joshgodsiff** [comment]
👍

**Pull request merged**